lib/creditsafe/request/find_company.rb

```diff
@@ -34,21 +37,12 @@ def message
 
       private
 
-      attr_reader :country_code, :registration_number, :city
+      attr_reader :country_code, :registration_number, :city, :company_name
 
       def check_search_criteria(search_criteria)
-        if search_criteria[:country_code].nil?
-          raise ArgumentError, "country_code is a required search criteria"
-        end
-
-        if search_criteria[:registration_number].nil?
-          raise ArgumentError,
-                "registration_number is a required search criteria"
-        end
-
-        if search_criteria[:city] && search_criteria[:country_code] != 'DE'
-          raise ArgumentError, "city is only supported for German searches"
-        end
+        raise ArgumentError, "country_code is a required search criteria" if search_criteria[:country_code].nil?
+        raise ArgumentError, "registration_number or company_name are required search criteria" if search_criteria[:registration_number].nil? && search_criteria[:company_name].nil?
```

Comment:
Good question. Haven't tested if it's possible to submit both. I'll try on Monday.